<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projectile_edit</class>
 <widget class="QWidget" name="projectile_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_8"/>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_10">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="3" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Name: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="projectileList_combo"/>
     </item>
     <item row="11" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="projectileDestructibleCheckBox">
         <property name="text">
          <string>Destructible</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>Hit Points (HP):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="projectileHitPointsSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="explosive_checkBox">
         <property name="text">
          <string>Explosive</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="vanishOnHit_checkBox">
         <property name="text">
          <string>Vanishes on Hit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSpinBox" name="img_w">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>320</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>Image Height:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="img_h">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>240</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>Preview:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Max Shots:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Trajectory Type:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="name">
       <property name="text">
        <string>No Name</string>
       </property>
       <property name="maxLength">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Add Projectile</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>Image Widht:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Image File:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="trajectory"/>
     </item>
     <item row="14" column="1">
      <widget class="QScrollArea" name="scrollArea_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="widgetResizable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>434</width>
          <height>558</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>100</horstretch>
          <verstretch>100</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <widget class="projectilePreviewArea" name="projectilePreviewAreaWidget" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>651</width>
           <height>581</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="graphic_filename"/>
     </item>
     <item row="12" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QSpinBox" name="max_shots"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Speed:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="speed"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Damage:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="damage"/>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sound Effect File:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="sfxFilename_comboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Selected:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>projectilePreviewArea</class>
   <extends>QWidget</extends>
   <header>projectilepreviewarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
